nemo_text_processing/inverse_text_normalization/es/taggers/time.py

```diff
@@ -171,14 +204,14 @@ def __init__(self):
             + delete_extra_space
             + pynutil.insert("minutes: \"")
             + delete_space
-            + pynutil.delete("menos")
+            + pynutil.delete(ES_MINUS)
             + delete_space
             + pynini.union(
-                pynini.cross("cinco", "55"),
-                pynini.cross("diez", "50"),
-                pynini.cross("cuarto", "45"),
-                pynini.cross("veinte", "40"),
-                pynini.cross("veinticinco", "30"),
+                pynini.cross(pynini.union("cinco", "Cinco"), "55"),
+                pynini.cross(pynini.union("diez", "Diez"), "50"),
+                pynini.cross(pynini.union("cuarto", "Cuarto"), "45"),
+                pynini.cross(pynini.union("veinte", "Veinte"), "40"),
+                pynini.cross(pynini.union("veinticinco", "Veinticinco"), "35"),
```

Comment:
Also add the informal version `veinte y cinco`. It's probably unnecessary but no harm for ITN.

Reply:
probably better not to, considering that we don't want erroneous ITN for an input like "Of these apples I got twenty and five of the other ones" -- sounds odd in English, but likely to exist in Spanish